Newsletters/_posts/Yearn-Finance-Newsletter-53/2022-01-11-Yearn-Finance-Newsletter-53.md

```diff
@@ -4,91 +4,91 @@ title: "Yearn Finance Newsletter #53"
 categories: [Newsletters]
 image: ./Yearn-Finance-Newsletter-53/image1.png
 author: Yearn
-translator:
-publish: false
+translator: RestlessMik3
+publish: true
 ---
 
-# Yearn Finance Newsletter #53
+# Newsletter de Yearn Finance #53
 
-### Week Ending January 9nd, 2022
+### Semana que finaliza el 9 de enero de 2022
 
-![](image1.png)
+![](imagen1.png)
```

Comment:
Don't translate this as the file name it is referencing is still image 1 this the image will then not show up.